ui/lib/fetch.js

```diff
@@ -1,6 +1,6 @@
 const fetch = global.fetch
 
-const base = '0.19.1'
+const base = '0.21.0'
```

Comment:
This isn't ideal. It's setting the API version to some version in the future which may cause problems down the road